src/engine/sys/manifests/enable-visual-styles.manifest.cmake.in

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
```

Comment:
For nice visual styles of UI controls.

Requires testing on Windows XP, if still supported.

Notice blue button border and new styles:
![image](https://user-images.githubusercontent.com/1983411/116013316-3d320600-a638-11eb-94c5-ceff092b359e.png)

Reply:
We don't support Windows XP, well at least, we don't care about building for it. It looks like no one tried to build it on/for XP so we don't know if it works, but at the same time it means we don't need to care about it. =)

Reply:
I guess that missing pak is intentionnally to trigger that error message, but anyway if you need some 0.52 data to test the build, you can download a release candidate build there: https://forums.unvanquished.net/viewtopic.php?f=4&t=2221

You would just have to do:

```sh
./daemon -pakpath path/to/pkg
```